aries-site/src/layouts/main/Layout.js

```diff
@@ -71,6 +70,22 @@ export const Layout = ({
     { id: 'main', label: 'Main Content' },
   ].filter(link => link !== undefined);
 
+  const [activeHeading, setActiveHeading] = useState();
+  const activeHeadingContextValue = React.useMemo(() => {
+    const nextValue = { activeHeading, setActiveHeading };
+    return nextValue;
+  }, [activeHeading]);
+
+  // useEffect finds which page heading is on screen as the user scrolls
+  useEffect(() => {
```

Comment:
I would definitely think this needs dependencies rather than always replacing the window.onscroll after each render.  I'd also think you'd use `window.addEventListener` instead so you're not affecting any other listeners? Something like:
```jsx
useEffect(() => {
  const onScroll = () => {
    if (window.pageYOffset === 0) {
      setActiveHeading(null);
    }
  };
  window.addEventListener('scroll', onScroll);
  return () => {
    window.removeEventListener('scroll, onScroll);
  };
}, [setActiveHeading]);
```